doc/source/user_guide/groupby.rst

```diff
@@ -31,29 +31,29 @@ following:
     * Filling NAs within groups with a value derived from each group.
 
 * **Filtration**: discard some groups, according to a group-wise computation
-  that evaluates True or False. Some examples:
+  that evaluates as True or False. Some examples:
 
-    * Discard data that belongs to groups with only a few members.
+    * Discard data that belong to groups with only a few members.
     * Filter out data based on the group sum or mean.
 
 Many of these operations are defined on GroupBy objects. These operations are similar
-to the :ref:`aggregating API <basics.aggregate>`, :ref:`window API <window.overview>`,
-and :ref:`resample API <timeseries.aggregate>`.
+to those of the :ref:`aggregating API <basics.aggregate>`,
+:ref:`window API <window.overview>`, and :ref:`resample API <timeseries.aggregate>`.
 
 It is possible that a given operation does not fall into one of these categories or
 is some combination of them. In such a case, it may be possible to compute the
 operation using GroupBy's ``apply`` method. This method will examine the results of the
-apply step and try to return a sensibly combined result if it doesn't fit into either
-of the above two categories.
+splitting step and try to return a sensibly combined result if it doesn't fit into either
+of the above three categories.
 
 .. note::
 
-   An operation that is split into multiple steps using built-in GroupBy operations
-   will be more efficient than using the ``apply`` method with a user-defined Python
+   An operation that is split into multiple steps using built-in GroupBy operations,
+   will be more efficient than one using the ``apply`` method with a user-defined Python
```

Comment:
I think the options are either:
"An operation that is split into multiple steps using built-in GroupBy operations, will be more efficient than one using the apply method with a user-defined Python function."

Or:

"Splitting into multiple steps using built-in GroupBy operations, will be more efficient than using the apply method with a user-defined Python function."

Reply:
I didn't notice the comma added here - I believe that is incorrect. These are not independent clauses.

In your second option above, I believe you're missing a noun: "Splitting an operation into multiple groups...". I see no reason to prefer one version over the other and because of that I think this should be left as is - but let me know if you think there is.

Reply:
OK thanks